debian/clearwater-monit.service

```diff
@@ -4,7 +4,7 @@
 Description=Clearwater-monit service manager
 
 [Service]
-ExecStartPre=/usr/share/clearwater/bin/issue-alarm "systemd" "4500.6"
+ExecStartPre=[ -x /usr/share/clearwater/bin/issue-alarm ] && /usr/share/clearwater/bin/issue-alarm "systemd" "4500.6" || true
```

Comment:
We discussed. `issue-alarm` is installed by clearwater-infrastructure which depends on clearwater monit for some reason. This feels like we've got our dependencies wrong - I would expect clearwater infrastructure to be at the "bottom" of the dependency graph, and I can't immediately think of anything in clearwater infrastructure that uses monit. You are going to check the code in as-is and investigate the dependency issue separately.